src/google_heatmap.js

```diff
@@ -1,21 +1,15 @@
-import fp from 'lodash/fp';
-
 export const generateHeatmap = (instance, { positions }) =>
   new instance.visualization.HeatmapLayer({
-    data: fp.reduce(
+    data: positions.reduce(
       (acc, { lat, lng }) => {
         acc.push({
           location: new instance.LatLng(lat, lng),
         });
         return acc;
       },
-      [],
-      positions
+      []
     ),
   });
 
 export const optionsHeatmap = (instance, { options }) =>
-  fp.map(
-    option => instance.set(option, options[option]),
-    Object.keys(options || {})
-  );
+  Object.keys(options).map(option => instance.set(option, options[option]));
```

Comment:
I don't know why but i am not able to run entire project, so everything looks good to me just map over an empty object as well if there is no options, like below.

```Object.keys(options || {}).map(option => instance.set(option, options[option]));```

Reply:
@ZAKdev Done, instead of doing what you said I've set a default value for the parameter. LMK How that works. I am merging this as for now.